bicycleparameters/cycle_app.py

```diff
@@ -69,7 +74,7 @@
                   'Steer Axis Tilt [degrees]:',
                   'Gravity [N/kg]:']
 
-app = dash.Dash(__name__, external_stylesheets=[dbc.themes.CYBORG])
+app = dash.Dash(__name__, external_stylesheets=[dbc.themes.MINTY])
```

Comment:
Let's keep the theming the same for this PR. If you want to open another PR for changing colors and such, you are welcome to do that.